src/seedu/addressbook/data/person/Person.java

```diff
@@ -25,13 +28,21 @@ public Person(Name name, Phone phone, Email email, Address address, UniqueTagLis
         this.email = email;
         this.address = address;
         this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        
+        this.sequenceNumber = nextSequenceNumber;
+        Person.nextSequenceNumber += 1;
     }
 
     /**
      * Copy constructor.
      */
     public Person(ReadOnlyPerson source) {
         this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getTags());
+        //unsure if should implement sequenceNumber in ReadOnlyPerson interface.
+    }
+    
+    public int getSequence(){
+        return sequenceNumber;
     }
```

Comment:
You can put this method in the `ReadOnlyPerson` interface and override it in this class. The variable can remain in this class.